app/deepseg.cc

```diff
@@ -630,9 +634,53 @@ int main(int argc, char* argv[]) try {
 
 		cv::Mat test;
 		cv::cvtColor(raw,test,cv::COLOR_YUV2BGR_YUYV);
-		char status[80];
-		snprintf(status, sizeof(status), "MainFPS: %5.2f AiFPS: %5.2f", mfps, afps);
-		cv::putText(test, status, cv::Point(5,test.rows-5), cv::FONT_HERSHEY_PLAIN, 1.0, cv::Scalar(0,255,0));
+		// frame rates at the bottom
+		if (showFPS) {
+			char status[80];
+			snprintf(status, sizeof(status), "MainFPS: %5.2f AiFPS: %5.2f", mfps, afps);
+			cv::putText(test, status, cv::Point(5,test.rows-5), cv::FONT_HERSHEY_PLAIN, 1.0, cv::Scalar(0,255,255));
+		}
+		// keyboard help
+		if (showHelp) {
+			static const char *help[] = {
+				"Keyboard help:",
+				" q: Quit",
+				" s: Switch filter on/off",
```

Comment:
Consistent upper/lower case …

Reply:
I was trying to indicate why the letters are chosen, not sure this helps :question: